tests/main/views/test_marketplace.py

```diff
@@ -1013,6 +1013,15 @@ def test_catalogue_of_briefs_page_404_for_framework_that_does_not_exist(self):
         assert res.status_code == 404
         self._data_api_client.find_frameworks.assert_called_once_with()
 
+    def test_briefs_search_does_not_have_js_hidden_filter_button(self):
+        res = self.client.get('/digital-outcomes-and-specialists/opportunities')
+        assert res.status_code == 200
+
+        document = html.fromstring(res.get_data(as_text=True))
+
+        filter_button = document.xpath('//button[@class="button-save"]')
```

Comment:
Yep, the equality check here tests the entire class string. To match subsets you have to use `contains(@class, 'button-save')`.